frontend/catalyst/jit.py

```diff
@@ -317,6 +323,27 @@ def _get_workspace(self):
 
         return WorkspaceManager.get_or_create_workspace(workspace_name, preferred_workspace_dir)
 
+    def _patched_run_autograph(self):
+        # Create a tuple of included modules to convert
+        include_convertlist = ()
+        if self.compile_options.autograph_include is not None:
+            for rule in self.compile_options.autograph_include:
+                include_convertlist = (ag_config.Convert(rule), *include_convertlist)
+
+        # Patch the autograph conversion rules with the included modules first
+        # and the block list second
+        # TODO: eliminate the need for patching by improving the autograph interface
+        with Patcher(
+            (ag_api, "_TRANSPILER", catalyst.autograph.TRANSFORMER),
+            (
+                ag_config,
+                "CONVERSION_RULES",
+                include_convertlist + ag_primitives.module_allowlist,
+            ),
+            (ag_primitives.ag_py_builtins, "BUILTIN_FUNCTIONS_MAP", ag_primitives.py_builtins_map),
+        ):
```

Comment:
How about patching just the `module_allowlist` from here?

Reply:
Done! Thanks for the hints on tackling this!